File: src/main/java-graphql/io/github/graphql/model/CodesOfConductQueryResponse.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class CodesOfConductQueryResponse extends GraphQLResult<Map<String, java.util.List<CodeOfConductTO>>> {

    private static final String OPERATION_NAME = "codesOfConduct";

    public CodesOfConductQueryResponse() {
    }

    public java.util.List<CodeOfConductTO> codesOfConduct() {
        Map<String, java.util.List<CodeOfConductTO>> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}

File: src/main/scala-graphql/io/github/graphql/model/CodesOfConductQueryResponse.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
class CodesOfConductQueryResponse extends GraphQLResult[JMap[String, Seq[CodeOfConductTO]]] {

    def codesOfConduct(): Seq[CodeOfConductTO] = {
        val data: JMap[String, Seq[CodeOfConductTO]] = getData
        if (data != null) data.get(CodesOfConductQueryResponse.OPERATION_NAME) else null
    }

}

object CodesOfConductQueryResponse {

    private final val OPERATION_NAME: String = "codesOfConduct"

}
